plugins/LilyPad/LilyPad.cpp

```diff
@@ -531,6 +531,8 @@ void Update(unsigned int port, unsigned int slot)
     dm->Update(&info);
     static int rapidFire = 0;
     rapidFire++;
+    static bool anyDevActiveAndBound = true;
+    bool currentDevActiveAndBound = false;
```

Comment:
No, it shouldn't be moved as a new pad can (dis)connect at other times than just when PADopen is triggered.

I actually used dev because of the code below `Device *dev`, but I've changed it to avoid the confusion.